releasenotes/notes/drop-gateway-alpha.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-managment
+releaseNotes:
+- |
+  **Removed** `alpha` Gateway API types by default. These can be explicitly re-enabled with `PILOT_ENABLE_ALPHA_GATEWAY_API=true`.
```

I think `They` works better here, or maybe `These types`.
```suggestion
  **Removed** `alpha` Gateway API types by default. They can be explicitly re-enabled with `PILOT_ENABLE_ALPHA_GATEWAY_API=true`.
```